Formula/p/pciutils.rb

```diff
@@ -0,0 +1,19 @@
+class Pciutils < Formula
+  desc "PCI utilities"
+  homepage "https://github.com/pciutils/pciutils"
+  url "https://github.com/pciutils/pciutils/archive/refs/tags/v3.10.0.tar.gz"
+  sha256 "e579d87f1afe2196db7db648857023f80adb500e8194c4488c8b47f9a238c1c6"
+  license "GPL-2.0-or-later"
+
+  uses_from_macos "zlib"
+
+  def install
+    system "make", "ZLIB=yes", "DNS=yes", "SHARED=yes", "PREFIX=#{prefix}", "MANDIR=#{man}"
+    system "make", "install", "ZLIB=yes", "DNS=yes", "SHARED=yes", "PREFIX=#{prefix}", "MANDIR=#{man}"
+    system "make", "install-lib", "ZLIB=yes", "DNS=yes", "SHARED=yes", "PREFIX=#{prefix}", "MANDIR=#{man}"
+  end
+
+  test do
+    assert_match "setpci version", shell_output("#{sbin}/setpci --version")
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
`pciutils` [doesn't support Apple Silicon](https://github.com/pciutils/pciutils/issues/111), so I can't actually test it. And even though it works, the output is tied to the hardware and is not the same between different hosts.

Reply:
You can do:
```rb
on_macos do
  depends_on arch: :x86_64
end
```